src/mixins/object_interactivity.mixin.js

```diff
@@ -194,11 +179,46 @@
         width,
         height
       );
+      hasControls && this.drawControlsConnectingLines(ctx);
 
       ctx.restore();
       return this;
     },
 
+    /**
+     * Draws lines from a borders of an object's bounding box to controls that have `withConnection` property set.
+     * Requires public properties: width, height
+     * Requires public options: padding, borderColor
+     * @param {CanvasRenderingContext2D} ctx Context to draw on
+     * @return {fabric.Object} thisArg
+     * @chainable
+     */
+    drawControlsConnectingLines: function (ctx) {
```

Comment:
doesn't respect group

Reply:
https://github.com/fabricjs/fabric.js/blob/69020dfaeadf8c1b15b31778a0ad0238f46106d2/src/mixins/object_interactivity.mixin.js#L248
this is wrong and I can't get it fixed.
It should not skew the point and it should scale it to account for the skew.
I tried something but I can't get the translation to correct:
https://github.com/fabricjs/fabric.js/blob/9d8fce4a568efc1dafabaceb69017aba30777453/src/mixins/object_interactivity.mixin.js#L228